### PR TITLE
Remove unnecessary permissions

### PR DESCRIPTION
**Pull request by ycao56**
Remove permissions on policies.ibm.com API group as they are not needed
by the controller.
Fixes https://github.com/stolostron/backlog/issues/17475
Signed-off-by: Yu Cao <ycao@redhat.com>

### CODE REVIEW

File: test/e2e/e2e_suite_test.go [hunk 1]
```diff
@@ -77,8 +77,10 @@ var _ = BeforeSuite(func() {
 			},
 		}, metav1.CreateOptions{})).NotTo(BeNil())
 	}
-	By("Create trustedcontainerpolicy CRD")
-	_, err := utils.KubectlWithOutput("apply", "-f", "../resources/policies.ibm.com_trustedcontainerpolicies_crd.yaml")
+	By("Create configpolicy CRD")
+	_, err := utils.KubectlWithOutput("apply", "-f",
+		"https://raw.githubusercontent.com/stolostron/config-policy-controller/"+
+			"main/deploy/crds/policy.open-cluster-management.io_configurationpolicies.yaml")
```
Comment (JustinKuli) [inline test/e2e/e2e_suite_test.go:83]: What do you think of making this point to the release-2.5 branch? Then if we ever run these tests against old commits, they would use the CRDs for that release, as opposed to whatever is the newest CRD.

Comment (ycao56) [reply test/e2e/e2e_suite_test.go:83]: Agreed. But we should change it in 2.5 branch so that main is always pointed to main.

Comment (ycao56) [reply test/e2e/e2e_suite_test.go:83]: Let me move this logic to makefile so that we can parameterize it

Comment (ycao56) [reply test/e2e/e2e_suite_test.go]: done

**Review by JustinKuli — APPROVED** [review]
/lgtm

**Pull request merged**